sdk/core/azure-core/src/samples/java/com/azure/core/util/ContextJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Code snippets for {@link Context}
+ */
+public class ContextJavaDocCodeSnippets {
+
+    /**
+     * Code snippet for {@link Context#Context(Object, Object)}
+     */
+    public void constructContextObject() {
```

Comment:
I'm not sure if we have any well known `Context` values that get passed through other than tracing, they would be good to use in these samples as they demonstrate real world scenarios.

Reply:
I added tracing relevant Context object uses in a few places. Since these were code snippets and outside of the tracing package could not have a complete scenario-based snippet.